src/util/get-github-client-config.ts

```diff
@@ -66,8 +63,8 @@ export async function createAppClient(logger: Logger, jiraHost: string): Promise
  * Factory function to create a GitHub client that authenticates as the installation of our GitHub app to get
  * information specific to an organization.
  */
-export async function createInstallationClient(githubInstallationId: number, jiraHost: string, logger: Logger): Promise<GitHubInstallationClient> {
-	const gitHubClientConfig = await getGitHubClientConfig(jiraHost);
+export async function createInstallationClient(githubInstallationId: number, logger: Logger, jiraHost: string): Promise<GitHubInstallationClient> {
```

Comment:
Okay, just see this line. `Move jiraHost so that it is the last param: this is currently only used to set the FF and will later be removed so makes more sense to pass it as the final argument wherever we're calling these functions.`

If it is related to that comment, can we still figure out a way? Not sure, but happy to discuss more.

Reply:
These functions were created as part of the GHE track so we can use a FF to determine what to call the GH clients with the additional params or not. The FF is currently off so existing code isn't affected by this. The args being passed in only take effect once we start rolling out the FF. It is possible to expect that there may be further changes to these factory functions params as we continue to work on this project but as long as the FF is off, we don't need to be concerned about making trivial changes such as this.